### DISCUSSION TITLE
release process question

### DISCUSSION THREAD

**Discussion by chenrui333** [prest/prest#397] [API Server]
👋 I have some questions about the repo's release process:

- **latest tag**, I saw the most recent latest tag is `1.3.0-test`, which is kind of weird, do you mind explaining it a little bit?
- **tagging**, another thing is the tagging process, I kind of running into issue when trying to keep the homebrew formula up to date (certainly there is retagging for the latest release, `1.3.0` at some point which caused [the checksum mismatch](https://github.com/Homebrew/homebrew-core/pull/59724)),  there is absolutely no issue from brew side to fix it though, but just curious if I can help with anything in here.

In general, happy to help. Thanks. :)

**Comment by avelino** [comment]
We were having trouble generating an automated release and were generating some testing between yesterday and today, sorry for the inconvenience and confusion.

you can see more here https://github.com/prest/prest/pull/396

**Comment by chenrui333** [comment]
I see, make sense. Thanks @avelino!
